doc/src/manual/documentation.md

```diff
@@ -568,7 +569,7 @@ A paragraph containing a **bold** word.
 Surround words with one asterisk, `*`, to display the enclosed text in italics.
 
 ```
-A paragraph containing an *emphasized* word.
+A paragraph containing an *italicized* word.
```

Comment:
The rendering of emphasis depends on the viewer. It happens to be italicized in the REPL, but that's not guaranteed.

Reply:
Though then again, the section title is literally italics, so maybe this is fine and non-italicized rendering should just be documented as a quirk of that renderer.